code/modules/clothing/spacesuits/chronosuit.dm

```diff
@@ -103,6 +105,7 @@
 			camera.remove_target_ui()
 			camera.forceMove(user)
 		teleport_now.UpdateButtonIcon()
+		deactivate(TRUE)
```

Comment:
This should most certainly not happen as the suit will turn off every time you teleport.

Reply:
This is intended, because otherwise it'd change perspective again and bug out. It's why I decreased cooldown, so that you can pretty much still use it right after.